src/Hosting.Services/HostBuilderExtensions.cs

```diff
@@ -41,6 +41,7 @@ public static class HostBuilderExtensions
 		/// <summary>
 		/// Registering Dependency Injection classes that will provide Service Fabric specific information for logging
 		/// </summary>
+		[Obsolete($"This method also adds Legacy OmexLogger and ActivityEventSender and they are deprecated. This legacy telemetry is pending for removal by 1 July 2024. Consider adding a different telemetry solution. Code: 8913598")]
```

Comment:
Obsoletion only makes sense for methods directly called by consumer code. Also don't forget that this is a public repo, don't add internal details.

Reply:
Hey, I mark them in some other places in case there is a contributor wants to use the code. 
Also, the code 8913598 is naive to external audience, but it helps us for better tracking in the future change.